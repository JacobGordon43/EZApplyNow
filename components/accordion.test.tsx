import { render, screen } from "@testing-library/react"
import {describe, expect, test, it} from '@jest/globals';
import FormDropdown from "./Accordion";
import PersonalForm from "./forms/personalForm";

describe('Home page', () => {
    it("should render", ()=>{
        const container = render(<FormDropdown value="personalForm" text="Personal Information"><PersonalForm /></FormDropdown>)
        
        expect(container).toMatchSnapshot();
    })
});


