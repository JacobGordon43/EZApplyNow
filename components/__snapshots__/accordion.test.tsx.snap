// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="w-11/12 max-w-[800px] m-auto"
        data-orientation="vertical"
      >
        <div
          data-orientation="vertical"
          data-state="closed"
        >
          <h3
            class="my-5 py-2 w-full bg-[#2DC653]"
            data-orientation="vertical"
            data-state="closed"
          >
            <button
              aria-controls="radix-:r1:"
              aria-expanded="false"
              class="AccordionTrigger flex w-full"
              data-orientation="vertical"
              data-radix-collection-item=""
              data-state="closed"
              id="radix-:r0:"
              type="button"
            >
              <span
                class="pl-2 float-right grow text-left"
              >
                Personal Information
              </span>
              <svg
                class="lucide lucide-chevron-down AccordionChevron w-14"
                fill="none"
                height="24"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="m6 9 6 6 6-6"
                />
              </svg>
            </button>
          </h3>
          <div
            aria-labelledby="radix-:r0:"
            class="AccordionContent"
            data-orientation="vertical"
            data-state="closed"
            hidden=""
            id="radix-:r1:"
            role="region"
            style="--radix-accordion-content-height: var(--radix-collapsible-content-height); --radix-accordion-content-width: var(--radix-collapsible-content-width);"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="w-11/12 max-w-[800px] m-auto"
      data-orientation="vertical"
    >
      <div
        data-orientation="vertical"
        data-state="closed"
      >
        <h3
          class="my-5 py-2 w-full bg-[#2DC653]"
          data-orientation="vertical"
          data-state="closed"
        >
          <button
            aria-controls="radix-:r1:"
            aria-expanded="false"
            class="AccordionTrigger flex w-full"
            data-orientation="vertical"
            data-radix-collection-item=""
            data-state="closed"
            id="radix-:r0:"
            type="button"
          >
            <span
              class="pl-2 float-right grow text-left"
            >
              Personal Information
            </span>
            <svg
              class="lucide lucide-chevron-down AccordionChevron w-14"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m6 9 6 6 6-6"
              />
            </svg>
          </button>
        </h3>
        <div
          aria-labelledby="radix-:r0:"
          class="AccordionContent"
          data-orientation="vertical"
          data-state="closed"
          hidden=""
          id="radix-:r1:"
          role="region"
          style="--radix-accordion-content-height: var(--radix-collapsible-content-height); --radix-accordion-content-width: var(--radix-collapsible-content-width);"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
